compiler/rustc_span/src/source_map.rs

```diff
@@ -336,7 +336,7 @@ impl SourceMap {
         mut file_local_non_narrow_chars: Vec<NonNarrowChar>,
         mut file_local_normalized_pos: Vec<NormalizedPos>,
         original_start_pos: BytePos,
-        original_end_pos: BytePos,
+        _original_end_pos: BytePos,
```

Comment:
Should we delete this argument completely? I could see us asserting start to end is source_len, I guess.